changelogs/unreleased/2704-incorrect-log-message.yml

```diff
@@ -0,0 +1,4 @@
+description: The log message is now consistent with what is happening
```

Comment:
This changelog message is quite vague. Could you add a more specific description about which log line was incorrect. Changelog messages are intended to be read by end-user, so they need a bit more context.